### DISCUSSION TITLE
1.2.4

### DISCUSSION THREAD

**Discussion by xiaye13579** [xiaye13579/BBLL#137] [Announcements]
1、修复番剧/影视选集的bug
2、修复追番/追剧无法进入番剧/影视详情页的bug

<hr /><em>This discussion was created from the release <a href='https://github.com/xiaye13579/BBLL/releases/tag/v1.2.4'>1.2.4</a>.</em>

**Comment by windhuang** [comment]
这节奏，太强了，新年还会放一个大招吗？

**Comment by mfirewire** [comment]
麻烦修复一下不能显示cc字幕的bug吧，感谢！
